demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -150,14 +150,14 @@ export function ProductsGrid() {
             renderCell: (params) => <>{params.row.tags.map((tag) => tag.title).join(", ")}</>,
             filterOperators: [
                 {
-                    value: "contains",
+                    label: "Search",
```

Comment:
Why not contains (label) as in other text fields?

Reply:
The contains logic is completely different than the search logic, that's why i used a different label. But the user might not see the difference

Reply:
I believe contains is better

On a side note, using contains for ID matching in ManyToManyFilter probably wasn't the best idea. Why didn't we use isAnyOf (as in ManyToOne)?

Reply:
> I believe contains is better

> On a side note, using contains for ID matching in ManyToManyFilter probably wasn't the best idea. Why didn't we use isAnyOf (as in ManyToOne)?

My thought might have been that the value is an array and contains searches thru that array. Is that not logical for you?

Reply:
It is logical to me, but now we have the name clash with the search through the fields. Maybe that's why MUI chose isAnyOf? 🤔

Reply:
So you suggest to:
- ManyToManyFilter: change from contains to isAnyOf
- ManyToManyFilter: rename the new search to contains

Reply:
That's what I initially suggested, yes

But I'm not sure anymore, since I've just now notice that OneToManyFilter.isAnyOf is for multiple IDs, and ManyToMany.contains is for a single ID (we could change that however)